.stylelintrc

```diff
@@ -11,17 +11,12 @@
     "@blueprintjs/no-color-literal": [true, { "disableFix": true }],
     "color-function-notation": "legacy",
     "declaration-empty-line-before": null,
-    "indentation": [2, {
-      "ignore": ["value"]
-    }],
-    "max-line-length": [
-      100,
-      { "ignore": ["comments"] }
-    ],
     "no-invalid-position-at-import-rule": [true, {
         "ignoreAtRules": ["use"]
     }],
     "scss/media-feature-value-dollar-variable": null,
-    "selector-max-universal": 1
+    "selector-max-universal": 1,
```

Comment:
Note that we should be able to remove these rules in https://github.com/palantir/blueprint/pull/7078 when we update `stylelint-config-palantir`.